std_msgs/Header header
    
uint16 statusSensor
uint32 id
uint16 age
uint8 statusMeasurement
uint8 statusMovement
uint16 positionInvalidFlags
uint8 position_Reference
float32 positionX
float32 positionXSTD
float32 positionY
float32 positionYSTD
float32 positionZ
float32 positionZSTD
float32 positionCovarianceXY
float32 positionOrientation
float32 positionOrientationSTD
uint8 existenceInvalidFlags
float32 existenceProbability
float32 existencePPV
uint8 classificationCar
uint8 classificationTruck
uint8 classificationMotorcycle
uint8 classificationBicycle
uint8 classificationPedestrian
uint8 classificationAnimal
uint8 classificationHazard
uint8 classificationUnknown
uint8 classificationOverdrivable
uint8 classificationUnderdrivable
uint8 dynamicsAbsVelInvalidFlags
float32 dynamicsAbsVelX
float32 dynamicsAbsVelXSTD
float32 dynamicsAbsVelY
float32 dynamicsAbsVelYSTD
float32 dynamicsAbsVelCovarianceXY
uint8 dynamicsRelVelInvalidFlags
float32 dynamicsRelVelX
float32 dynamicsRelVelXSTD
float32 dynamicsRelVelY
float32 dynamicsRelVelYSTD
float32 dynamicsRelVelCovarianceXY
uint8 dynamics_AbsAccel_InvalidFlags
float32 dynamicsAbsAccelX
float32 dynamicsAbsAccelXSTD
float32 dynamicsAbsAccelY
float32 dynamicsAbsAccelYSTD
float32 dynamicsAbsAccelCovarianceXY
uint8 dynamicsRelAccelInvalidFlags
float32 dynamicsRelAccelX
float32 dynamicsRelAccelXSTD
float32 dynamicsRelAccelY
float32 dynamicsRelAccelYSTD
float32 dynamicsRelAccelCovarianceXY
uint8 dynamicsOrientationInvalidFlags
float32 dynamicsOrientationRateMean
float32 dynamicsOrientationRateSTD
uint32 shapeLengthStatus
uint8 shapeLengthEdgeInvalidFlags
float32 shapeLengthEdgeMean
float32 shapeLengthEdgeSTD
uint32 shapeWidthStatus
uint8 shapeWidthEdgeInvalidFlags
float32 shapeWidthEdgeMean
float32 shapeWidthEdgeSTD